### PR TITLE
feat(ge): add verified checkmark to project details

### LINKED ISSUE #649: Add verified checkmark to GE project details
**User Story:**
As a donor
I want to see if a user has verified their social details
So that I know if they are legitimate

**Acceptance Criteria**
GIVEN that I am on the Grant Explorer project details page
WHEN a project has verified their social details
THEN I see the verified checkmark next to the relevant social info

**Product & Design Links:**
[Figma](https://www.figma.com/file/lZwHrKYiWJna3AePzyTAiG/Grant-Explorer?node-id=1583%3A56059)

**Tech Details:**

**Open Questions:**

**Notes/Assumptions:**
- This includes writing test coverage for the functionality added in the ticket, if necessary

### PR DESCRIPTION
**Pull request by chibie**
##### Description

This PR adds verified/unverified badges to the grant explorer project details

##### Refers/Fixes

closes #649

### CODE REVIEW

File: packages/grant-explorer/src/features/round/ViewProjectDetails.tsx [hunk 1]
```diff
@@ -109,6 +125,12 @@ function ProjectTitle(props: { projectMetadata: ProjectMetadata }) {
 }
 
 function AboutProject(props: { projectToRender: Project }) {
+  const [verifiedProviders, setVerifiedProviders] = useState<{
+    [key: string]: VerifiedCredentialState;
+  }>({
+    github: VerifiedCredentialState.PENDING,
+    twitter: VerifiedCredentialState.PENDING,
+  });
```
Comment (chibie) [inline packages/grant-explorer/src/features/round/ViewProjectDetails.tsx:133]: @thelostone-mc are these the only verifiable credentials for this page?

there's personal and company github, i'm not sure about the keys for these. is this something i'll have to ask grant hub folks?

File: packages/grant-explorer/src/features/round/ViewProjectDetails.tsx [hunk 2]
```diff
@@ -117,6 +139,62 @@ function AboutProject(props: { projectToRender: Project }) {
...
+      case VerifiedCredentialState.INVALID:
+        return (
+          <span className="rounded-full bg-red-100 px-3 inline-flex flex-row justify-center items-center">
+            <XCircleIcon
+              className="w-5 h-5 text-white mr-2"
+              data-testid={`${provider}-verifiable-credential-unverified`}
+            />
+            <p className="text-white font-medium text-xs">Invalid</p>
+          </span>
+        );
```
Comment (chibie) [inline packages/grant-explorer/src/features/round/ViewProjectDetails.tsx:192]: @nfrgosselin i included the state for unverified credentials to match application detail page in round manager. it's not explicitly in the figma design

should we keep this?

File: packages/grant-explorer/src/features/round/ViewProjectDetails.tsx (hunk 1 above)
Comment (thelostone-mc) [reply packages/grant-explorer/src/features/round/ViewProjectDetails.tsx:133]: Yup it looks like that!
@gravityblast Could you offer some guidance here ?

Comment (gravityblast) [reply packages/grant-explorer/src/features/round/ViewProjectDetails.tsx:133]: @chibie @thelostone-mc yes we have VC only for project's twitter and github

Comment (chibie) [reply packages/grant-explorer/src/features/round/ViewProjectDetails.tsx:133]: @gravityblast is it personal or project github?

Comment (gravityblast) [reply packages/grant-explorer/src/features/round/ViewProjectDetails.tsx:133]: @chibie it's verifying that your github account is part of the org you specified

**Review by melissa-neira — CHANGES_REQUESTED** [review]
Very small change (don't hate me😅): the horizontal padding for the verification chip should be 10px and I believe right now it's 12px. Don't ask me how I noticed this lol

**Comment by chibie** [comment]
@nfrgosselin @thelostone-mc  granthub has confirmed that we have a subgraph for project registry... i'll be changing the code to fetch from the subgraph before bringing this up for review again.

moving to draft now.

**Comment by chibie** [comment]
> Very small change (don't hate me😅): the horizontal padding for the verification chip should be 10px and I believe right now it's 12px. Don't ask me how I noticed this lol

this is done..haha ✅

**Review by melissa-neira — APPROVED** [review]

**Review by nfrgosselin — APPROVED** [review]

**Review by thelostone-mc — APPROVED** [review]

**Pull request merged**